gui/src/forms/AddModelForm.tsx

```diff
@@ -49,8 +50,15 @@ function AddModelForm({
       return false;
     }
 
-    const apiKey = formMethods.watch("apiKey");
-    return typeof apiKey === "undefined" || apiKey.length === 0;
+    const required = selectedProvider.collectInputFor
+      .filter((input) => input.required)
+      .map((input) => {
+        const value = formMethods.watch(input.key);
+        return value;
+      });
+    console.log("Required fields values:", required);
```

Comment:
```suggestion
    console.debug("Required fields values:", required);
```

We should document this somewhere but using `console.debug` logs to reduce noise

Reply:
done